dts/bindings/mtd/atmel,at45.yaml

```diff
@@ -45,6 +45,18 @@ properties:
     required: true
     description: Flash page size in bytes.
 
+  workaround-broken-read:
```

Comment:
workaround-broken-read-0x0b,  workaround-broken-read-h0b, workaround-broken-cmd-h0b, workaround-broken-opcode-h0b - maybe something more descriptive to what you are fixing. There are many read commands.

Reply:
updated the names, LMK what you think